apps/api/app/services/tracked_todo_service.py

```
@@ -0,0 +1,400 @@
+
+        Returns True if job was enqueued successfully, False otherwise.
+        """
+        try:
+            pool = await RedisPoolManager.get_pool()
+            await pool.enqueue_job(
+                "execute_tracked_todo",
+                todo_id,
+                _defer_until=scheduled_at,
+            )
+            return True
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)